test/functional/change_stream_spec.test.js

```diff
@@ -2,13 +2,30 @@
 
 const path = require('path');
 const chai = require('chai');
-const loadSpecTests = require('../spec').loadSpecTests;
+const { loadSpecTests } = require('../spec');
+const { runUnifiedTest } = require('./unified-spec-runner/runner');
```

Comment:
```suggestion
const { runUnifiedSuite } = require('./unified-spec-runner/runner');
```

If we go with the other suggested change